packages/mention/src/index.tsx

```diff
@@ -58,8 +59,10 @@ export interface MentionPluginConfig {
   mentionSuggestionsComponent?: ComponentType;
   entityMutability?: 'SEGMENTED' | 'IMMUTABLE' | 'MUTABLE';
   mentionTrigger?: string;
+  mentionTriggers?: string[];
```

Comment:
If we have now `mentionTrigger` and `mentionTriggers` what happens if you set both?

Reply:
- If you'd like to use only one trigger, then you should use mentionTrigger like above
`const mentionPlugin = createMentionPlugin({ mentionTrigger: '(' });`

- if you'd like to use multiple triggers, then you should use mentionTriggers like
`const mentionPlugin = createMentionPlugin({ mentionTriggers: ['@', '#'] });`

I think this makes more sense.

Reply:
can you change it to `mentionTrigger?: string | string[];` please?